src/styles/font-awesome-4.7.0/HELP-US-OUT.txt

```diff
@@ -0,0 +1,7 @@
+I hope you love Font Awesome. If you've found it useful, please do me a favor and check out my latest project,
+Fort Awesome (https://fortawesome.com). It makes it easy to put the perfect icons on your website. Choose from our awesome,
+comprehensive icon sets or copy and paste your own.
+
+Please. Check it out.
+
+-Dave Gandy
```

Comment:
You have 38 files added to your repository because of your use of font-awesome library. Instead of having the files locally, you can make use of a CDN and simply include the CDN link in your html.